### PR TITLE
Add question and occupied specific pieces 

### PR DESCRIPTION
**Pull request by Sonagh**

### CODE REVIEW

File: src/Chessboard/helpers.js [hunk 1]
```diff
@@ -186,6 +187,9 @@ function pieceCodeToFen(piece) {
   if (pieceCodeLetters[0] === 'w') {
     return pieceCodeLetters[1].toUpperCase();
   }
+  if (piece === '?') {
```
Comment (TigranOhanyan) [inline src/Chessboard/helpers.js:190]: what about the 0? I would like to see explicit matching for every case. That will make the code more readable (at least for me).
```suggestion
  if (piece === '?') {
  return '?';
  }
  if (piece === '0') {
  return '0';
  }
  if (pieceCodeLetters[0] === 'w') {
    return pieceCodeLetters[1].toUpperCase();
  }
  if (pieceCodeLetters[0] === 'b') {
    return pieceCodeLetters[1].toLowerCase();
  }
```

File: src/Chessboard/index.js [hunk 2]
```diff
@@ -21,8 +21,7 @@ const ChessboardContext = React.createContext();
 
 const getPositionObject = position => {
   if (position === 'start')
-    // return fenToObj('rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR');
-  return fenToObj('????R?r?/?????kq?/????Q???/????????/????????/????????/????????/????????');
+    return fenToObj('rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR');
```
Comment (TigranOhanyan) [inline src/Chessboard/index.js:24]: our initial position is all question marks

File: src/Chessboard/helpers.js [hunk 3]
```diff
@@ -136,12 +136,13 @@ function fenToPieceCode(piece) {
   if (piece === '-') {
     return '-';
```
Comment (TigranOhanyan) [inline src/Chessboard/helpers.js:137]: we don't have a piece or an icon for the hyphen. I think we need to somehow filter hyphens before calling this function

**Review by TigranOhanyan — COMMENTED** [review]
I would like to see your feedback about my review

File: src/Chessboard/index.js (hunk 2 above)
Comment (Sonagh) [reply src/Chessboard/index.js:24]: yes, but as we haven't delete question mark functionality yet, we can make this point with another step after creating valid fen functionality.

File: src/Chessboard/helpers.js (hunk 3 above)
Comment (Sonagh) [reply src/Chessboard/helpers.js:137]: ok, I removed it, increase it by mistake

File: src/Chessboard/index.js (hunk 2 above)
Comment (TigranOhanyan) [reply src/Chessboard/index.js:24]: In that case, let's open an issue
https://github.com/chessfinder/chessboardjsx/issues/3

File: src/Chessboard/helpers.js (hunk 1 above)
Comment (Sonagh) [reply src/Chessboard/helpers.js:188]: I think change 0 or ? to "occupied" and "unknown" text make code more complicated, cause at the and I will add the oder code that should do the opposite for sending you fen. In future we can add some tooltip on piece which can describe the piece.

**Review by TigranOhanyan — APPROVED** [review]

**Pull request merged**